pkg/controller/manager/types_old.go

```diff
@@ -19,13 +19,25 @@ import (
 	"github.com/scylladb/scylla-operator/pkg/util/duration"
 )
 
-type RepairTask scyllav1.RepairTaskStatus
+type startDateGetter interface {
+	GetStartDateOrEmpty() string
+}
+
+type startDateGetterSetter interface {
+	startDateGetter
+	SetStartDate(sd string)
+}
+
+type RepairTaskSpec scyllav1.RepairTaskSpec
+type RepairTaskStatus scyllav1.RepairTaskStatus
```

Comment:
given these are meant for implementing member methods the should be defined in the regular order (type+methods)

Reply:
done